api/src/main/java/org/openmrs/module/eptsreports/reporting/library/dimensions/EptsCommonDimension.java

```diff
@@ -132,6 +132,9 @@ public CohortDefinitionDimension age(AgeDimensionCohortInterface ageDimensionCoh
     dim.addCohortDefinition(
         DimensionKeyForAge.bellow2Years.getKey(),
         ageDimensionCohort.createXtoYAgeCohort("patients with age below 2 years", 0, 1));
+    dim.addCohortDefinition(
+        DimensionKeyForAge.overOrEqualTo2Years.getKey(),
+        ageDimensionCohort.createXtoYAgeCohort("patients with age below 2 years", 2, null));
```

Comment:
Sounds like it should be named **... over or equal to 2 years**
Please correct the name of the cohort